services-in-london/src/App.js

```diff
@@ -1,12 +1,78 @@
 import React, { Component } from 'react';
 import './App.css';
 import Area from './area';
-import Organization from './organization';
+import Type from './type';
+import Clients from './clients';
 
 class App extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
+				main: ['area','type','clients'],
+				clickedValue: null,
+				callData: null,
+		}
+	}
+	servicesBy = (info) =>{
+		this.setState({clickedValue: info});
+		this.componentDidMount(info);
```

Comment:
You should not need to call lifecycle methods, they happen during the component life cycle, just call the end function directly:
```javascript
this.callAPI(url);
```